src/toady/cli.py

```
@@ -189,14 +196,147 @@ def reply(ctx: click.Context, comment_id: str, body: str, pretty: bool) -> None:
+    action_symbol = "🔓" if undo else "🔒"
+
+    if pretty:
+        click.echo(f"{action_symbol} {action} thread {thread_id}")
+    else:
+        # For JSON output, we'll just return the result without progress messages
+        pass
+
+    # Execute the resolve/unresolve operation using the resolve service
+    try:
+        resolve_service = ResolveService()
+        
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)